### DISCUSSION TITLE
"The Initial Setup System is Disabled"

### DISCUSSION THREAD

**Discussion by PigmyRhino91** [Pryaxis/TShock#2092] [General]
New server owner here, trying to make myself an admin so i can start setting up permissions and spawn and stuff. 

I try to set it up and it says 

"THE INITIAL SETUP SYSTEM IS DISABLED. THIS INCIDENT HAS BEEN LOGGED.
IF YOU ARE LOCKED OUT OF ALL ADMIN ACCOUNTS, ASK FOR HELP ON HTTPS://TSHOCK.CO"

So that's what i'm doing, please advise

Thank you!

**Comment by hakusaro** [comment]
In this case, you need to take action to create a new account. You can use `/user add newuser newpassword owner` from the server console, and then use `/login newuser newpassword` to get access to your new account. Does that help?

**Reply by rekson1** [reply]
/user doesnt work and keeps saying "type /user for help", but it doesnt bring anything up
